src/components/button/button.svelte

```diff
@@ -203,13 +203,44 @@
       }
     }
   }
-  .leoButton.isCTA {
+  .leoButton.isHero {
     transition: var(--default-transition);
-    --bg: var(--gradient-call-to-action);
-    --bg-hover: var(--color-secondary-40);
-    --bg-active: var(--bg-hover);
+    --bg: transparent;
     --bg-focus: var(--bg);
     --bg-disabled: var(--color-gray-30);
     --color: white;
+    --default-bg-opacity: 1;
+
+    position: relative;
+    overflow: hidden;
```

Comment:
This makes me a little nervous as it could cut off shadows, right? It doesn't seem to have any negative effect at the moment though so could be ok.

Reply:
Good call, and easily fixed. https://github.com/brave/leo/pull/75/commits/528337ca155d9ba5df6f1baa4bd984b047a576ad